08-transport.Rmd

```diff
@@ -329,17 +359,22 @@ The following command makes use of the ability of simple features objects to con
 desire_carshort$geom_car = route_carshort$geometry
 ```
 
-This allows for the desire lines along which many short car journeys take place to be plotted alongside likely routes traveled by cars, with the width of the routes proportional to the number of car journeys that could potentially be replaced.
+This allows plotting the desire lines along which many short car journeys take place alongside likely routes traveled by cars, with the width of the routes proportional to the number of car journeys that could potentially be replaced.
 The code below results in Figure \@ref(fig:routes), which demonstrates along which routes people are driving short distances:
 
+<!--
+1. Why is the starting point for both routes different (shouldn't it be the polygon centroid)? Or the centroid is in the middle of nowhere. Then say so in the caption and/or use leaflet in the background.
```

Comment:
fig:routes remarks
1. Why is the starting point for both routes different (shouldn't it be the polygon centroid)? Or the centroid is in the middle of nowhere. Then say so in the caption and/or use leaflet in the background.
2. Caption: car journeys are made (red); As far as I understood you used polygon centroids as starting points. Then you computed the route between centroids. I wouldn't say that this route is representative or the most used route. It is just the shortest route between two centroids. Since we have only aggregated data on the polygon level, we simply have no clue what the most frequent point origins and destinations are. 
3. Plotting the centroids and/or the polygons would also help visual inspection.

Reply:
Happy to consider further updates but think this footnote addresses the main points: 

```
In this plot the origins of the red routes and black desire lines are not identical.
This is because zone centroids rarely lie on the route network: instead the route originate from the transport network node nearest the centroid.
Note also that routes are assumed to originate in the zone centroids, a simplifying assumption which is used in transport models to reduce the computational resources needed to calculate the shortest path between all combinations of possible origins and destinations [@hollander_transport_2016].
```

Reply:
Yes, perfect as is. And thanks for putting up with my comments!

Reply:
No problem, thanks for taking the time to make detailed comments, really useful internal review that will make the external review stronger.